src/main/resources/application.yml

```diff
@@ -12,7 +12,11 @@ application:
   domain: ${APP_DOMAIN:}
   email:
     sender: ${EMAIL_SENDER}
+    enabled: ${NOTIFICATIONS_EMAIL_ENABLED:false}
   environment: ${ENVIRONMENT:local}
+  in-app:
+    enabled: ${NOTIFICATIONS_IN_APP_ENABLED:false}
+  notifications-whitelist: ${NOTIFICATIONS_WHITELIST:[]}
```

Comment:
I included this as an env var, but it could be held as a Cognito group, as we do for the DSP pilot. That might be better, I'm not sure.

Reply:
Lets see how we end up using it.

Reply:
Lets see how we end up using it before we make it any more complicated.